plugins/wysiwygarea/plugin.js

```diff
@@ -716,3 +808,33 @@ CKEDITOR.config.disableNativeSpellChecker = true;
  * @param {CKEDITOR.editor} editor This editor instance.
  * @param {CKEDITOR.dom.element} data The element being added.
  */
+
+/**
+ * Document observation starting point. Native DOM object.
```

Comment:
The first line should say something about the purpose of this option, so I'd add a few words like `mutation observer` and `detach` 😛